dependencies/direct/py.in

```diff
@@ -4,6 +4,7 @@ covdefaults
 coverage  # accessed directly from tox
 coverage-enable-subprocess
 hypothesis
+Jinja2
```

Comment:
Looks like this is covering the side effect of the interfaces repo not declaring the dependency in the core packaging metadata. It might be a good idea to document that this is temporary.